functions-app/report-psychometrician-create-download/service/psychometrician-report.service.js

```diff
@@ -104,11 +106,17 @@ const psychometricianReportService = {
       throw error
     }
 
-    const arStat = await fs.stat(anomalyReportFilename)
-    this.logger.verbose(`${functionName}: anomaly report size: ${Math.round(arStat.size / 1024 / 1024)} MB`)
+    try {
+      const arStat = await fs.stat(anomalyReportFilename)
+      this.logger.verbose(`${functionName}: anomaly report size: ${Math.round(arStat.size / 1024 / 1024)} MB`)
+      filesToZip.push(anomalyReportFilename)
+    } catch (error) {
+      // Anomaly report may not have been produced if there weren't any anomlies
```

Comment:
Typo: `anomlies` should be `anomalies` 🙂

Reply:
Thanks! now fixed.